### PR TITLE
Update grafana

### PR DESCRIPTION
**Pull request by gabrielifg**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

bug fixes

* **What is the current behavior?** (You can also link to an open issue here)

Currently, the grafana version is out of date and contains vulnerability issues. Also some exporters, CAdvisor and node exporter have ports and "context deadline" issues presented by promethues in targets panel.

* **What is the new behavior (if this is a feature change)?**

The grafana version has been updated, and the CAdvisor and Node Exporter services are working as planned.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

dojot/dojot#2401

* **Other information**:

### CODE REVIEW

**Review by ewerton-silva00 — CHANGES_REQUESTED** [review]
You can remove the CAdvisor's ports publication. We will use these ports only in cases of troubleshooting.

File: monitoring/docker-compose-monitoring.yaml [hunk 1]
```diff
@@ -132,6 +132,11 @@ services:
     container_name: cadvisor
     image: gcr.io/cadvisor/cadvisor:v0.38.8
     privileged: true
+    ports:
```
[the remark this answers is not in the document]
Comment (gabrielifg) [reply monitoring/docker-compose-monitoring.yaml]: Okay, i will remove

**Review by ewerton-silva00 — APPROVED** [review]
Thank you! 👏🏼

File: docker-compose.yml [hunk 2]
```diff
@@ -1132,7 +1132,7 @@ services:
       - auth
       - device-manager
     ports:
-      - 3000:3000
+      - 3110:3000
```
Comment (danielapetito) [inline docker-compose.yml:1135]: A atualização do grafana precisa alterar porta do basic- auth ?

Pergunto pq aí precisa ajustar no K8S/Ansible.

Comment (gabrielifg) [reply docker-compose.yml:1135]: Sim Dani, analisei os dois serviços, alterar a porta do grafana seria mais custoso.

Comment (danielapetito) [reply docker-compose.yml:1135]: mas isso não terá impacto nos outros serviços ?
não seria interessante alinhar com o Gil @gilmaicor ?

**Review by danielapetito — APPROVED** [review]
ok

**Pull request merged**